src/pattern-library/components/PlaygroundApp.tsx

```diff
@@ -164,77 +151,88 @@ export default function PlaygroundApp({
   >;
   return (
     <Router base={baseURL}>
-      <main className="max-w-[1250px] mx-auto">
-        <div className="md:grid md:grid-cols-[2fr_5fr]">
-          <div className="md:h-screen md:sticky md:top-0 overflow-scroll">
-            <div className="md:sticky md:top-0 h-16 px-6 flex items-center bg-slate-0 border-b">
-              <h1 className="text-lg">
-                <Link href={baseURL + '/'} classes="grow flex gap-x-2">
-                  <LogoIcon />
-                  Component Library
-                </Link>
-              </h1>
+      <div className="w-full bg-stone-200">
+        <main className="max-w-[1300px] mx-auto">
+          <div className="md:grid md:grid-cols-[2fr_5fr]">
+            <div className="md:h-screen md:sticky md:top-0 border-l border-stone-400 bg-stone-100">
+              <ScrollContainer borderless>
+                <div className="h-16 flex items-center">
+                  <h1 className="font-medium text-lg pl-6">
+                    <Link
+                      href={baseURL + '/'}
+                      classes="flex gap-x-3 text-stone-700"
+                    >
+                      <LogoIcon className="text-brand" />
+                      Component Library
+                    </Link>
+                  </h1>
+                </div>
+                <Scroll>
+                  <ScrollContent classes="bg-stone-400/10">
```

Comment:
Tailwind's color palette is pretty awesome. I need this background to be mostly transparent so that the scroll shadows will show up. `stone-400/10` ("stone-400" color at 10% opacity) is exactly the same as `stone-100`, matching the header background. Sweet.